tests/integration/component.go

```diff
@@ -496,4 +510,26 @@ func componentTests(args ...string) {
 			helper.Chdir(originalDir)
 		})
 	})
+
+	Context("Creating component with numeric named context", func() {
+		var context string
```

Comment:
Already declared as global. You can remove it from here

Reply:
There is a reason for this here, as the test run parallel - this could get changed being a different context

Reply:
Hence a different variable then the global one

Reply:
Right

Reply:
Resolving